chrill-christella/mybrand-API
# Ft documentation

# What does this PR do? 
documenting the endpoints for testing

### Feature Description
This PR introduces a new feature called ft-documentation, which improves the documentation of the API. The feature includes the following change:
- A new documentation page that provides an overview of the API's functionality


### How to Test
-install dependencies by typing in the terminal npm i swagger-ui-express swagger-jsdoc 
- Start the application by running `npm start`
- Visit `http://localhost:3001/documentation` in your browser
- this will create the swagger ui
- Verify that the documentation page provides a clear and accurate overview of the application's functionality
-test all the endpoints including the authorized ones
